### PR TITLE
Changing physical constant values.

### PR DESCRIPTION
**Pull request by jlconlin**
Surprisingly, this changed the outputs in very significant ways. How do we know which one is correct?

### CODE REVIEW

**Comment by apmccartney** [comment]
Sorry for the delay. In which test+tapes did you find non-negligible changes?

File: src/phys.f90 [hunk 1]
```diff
@@ -14,20 +14,20 @@ module physics
...
-  real(kr),parameter,public::bk=8.617385e-5_kr
-  real(kr),parameter,public::amassn=1.008664904e0_kr
-  real(kr),parameter,public::amu=1.6605402e-24_kr
-  real(kr),parameter,public::hbar=1.05457266e-27_kr
-  real(kr),parameter,public::ev=1.60217733e-12_kr
+! real(kr),parameter,public::bk=8.617385e-5_kr
+! real(kr),parameter,public::amassn=1.008664904e0_kr
+! real(kr),parameter,public::amu=1.6605402e-24_kr
+! real(kr),parameter,public::hbar=1.05457266e-27_kr
+! real(kr),parameter,public::ev=1.60217733e-12_kr
```
Comment (apmccartney) [inline src/phys.f90]: why did ev change by 7 orders of magnitude?

Comment (kahlerac) [inline src/phys.f90]: Also, based upon what I started, but never completed for NJOY2012 a while ago there are also a number of particle mass constants to update ... broadr, heatr, groupr, thermr, leapr at least.

**Comment by apmccartney** [comment]
Following the electronVolt value correction, which test/tape combinations had significant changes?

**Comment by jlconlin** [comment]
With all of the constants changed, only tests 3,5, and 6 passed. I have not checked to see specifically why the others failed.

**Pull request closed without merging**